pages/fundamentals/telemetry.mdx

```diff
@@ -23,19 +23,33 @@ While a Memgraph database instance is running and an open internet connection is
 available, the following data will be sent to and stored on Memgraph's servers:
 
 - **Information about the host machine**
-    - CPU model
-    - Memory information
-    - Host OS
-    - Kernel information
-    
+  - CPU model
+  - Memory information
+  - Host OS
+  - Kernel information
+<br/>    
+
+- **Information about the clients**
+  - client name
+  - supported and agreed Bolt version
+  - number of sessions from each clients
+  - number of executed queries
+  - authentication type (user or anonymous)
```

Comment:
This is called connection type in our telemetry database, @gitbuda is authentication type ok?